extensions/arc/src/localizedConstants.ts

```diff
@@ -61,6 +61,8 @@ export const controllerEndpoint = localize('arc.controllerEndpoint', "Controller
 export const extensionName = localize('arc.extensionName', "Extension name");
 export const extensionsDescription = localize('arc.extensionsDescription', "PostgreSQL provides the ability to extend the functionality of your database by using extensions. Extensions allow for bundling multiple related SQL objects together in a single package that can be loaded or removed from your database with a single command. After being loaded in the database, extensions can function like built-in features.");
 export const extensionsFunction = localize('arc.extensionsFunction', "Some extensions must be loaded into PostgreSQL at startup time before they can be used. These preloaded extensions can be viewed and edited  below.");
+export const extensionsAddFunction = localize('arc.extensionsAddFunction', "Some extensions must be loaded into PostgreSQL at startup time before they can be used. To edit, type in comma separated list of valid extensions: citus, pgaudit, pgautofailover, pg_cron, pg_partman, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, tdigest.");
+export const extensionsAddErrorrMessage = localize('arc.extensionsAddErrorrMessage', "Value should be either of the following: citus, pgaudit, pgautofailover, pg_cron, pg_partman, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, tdigest.");
```

Comment:
If possible, it'd be nice to pass the supported list as an argument to the messages so it's only defined in 1 place.